boolean_constraints.tf

```diff
@@ -18,68 +18,68 @@
   Organization policy (boolean constraint)
  *****************************************/
 resource "google_organization_policy" "org_policy_boolean" {
-  count = "${local.organization && local.boolean_policy ? 1 : 0}"
+  count = local.organization && local.boolean_policy ? 1 : 0
 
-  org_id     = "${var.organization_id}"
-  constraint = "${var.constraint}"
+  org_id     = var.organization_id
+  constraint = var.constraint
 
   boolean_policy {
-    enforced = "${var.enforce}"
+    enforced = var.enforce
   }
 }
 
 /******************************************
   Folder policy (boolean constraint)
  *****************************************/
 resource "google_folder_organization_policy" "folder_policy_boolean" {
-  count = "${local.folder && local.boolean_policy ? 1 : 0}"
+  count = local.folder && local.boolean_policy ? 1 : 0
 
-  folder     = "${var.folder_id}"
-  constraint = "${var.constraint}"
+  folder     = var.folder_id
+  constraint = var.constraint
 
   boolean_policy {
-    enforced = "${var.enforce}"
+    enforced = var.enforce
   }
 }
 
 /******************************************
   Project policy (boolean constraint)
  *****************************************/
 resource "google_project_organization_policy" "project_policy_boolean" {
-  count = "${local.project && local.boolean_policy ? 1 : 0}"
+  count = local.project && local.boolean_policy ? 1 : 0
 
-  project    = "${var.project_id}"
-  constraint = "${var.constraint}"
+  project    = var.project_id
+  constraint = var.constraint
 
   boolean_policy {
-    enforced = "${var.enforce}"
+    enforced = var.enforce
   }
 }
 
 /******************************************
   Exclude folders from policy (boolean constraint)
  *****************************************/
 resource "google_folder_organization_policy" "policy_boolean_exclude_folders" {
-  count = "${local.boolean_policy && !local.project ? local.exclude_folders_list_length : 0}"
+  count = local.boolean_policy && ! local.project ? local.exclude_folders_list_length : 0
 
-  folder     = "${var.exclude_folders[count.index]}"
-  constraint = "${var.constraint}"
+  folder     = var.exclude_folders[count.index]
+  constraint = var.constraint
 
   boolean_policy {
-    enforced = "${!var.enforce}"
+    enforced = ! var.enforce
```

Comment:
change to var.enforce != true

Reply:
Thanks! This case isn't covered by any examples or tests.
Logged #12 for the issue. 
Addressed in [897c10d](https://github.com/terraform-google-modules/terraform-google-org-policy/pull/11/commits/897c10d3eb058ca9c1cd54b8f6273fa2b30ce99a)